jax/numpy/lax_numpy.py

```diff
@@ -3055,14 +3055,38 @@ def polyadd(a1, a2):
 def polyder(p, m=1):
```

Comment:
If we're requiring `m` to be concrete (as implied by the change from `arange` to `np.arange` below), then we should call
```python
m = core.concrete_or_error(int, m, "The error arose in jnp.polyder argument `m`")
```
Same in `polyint` below.